.github/workflows/cd_frontend.yml

```diff
@@ -0,0 +1,153 @@
+name: 'CD - Frontend'
+
+on:
+  push:
+    branches:
+      - master
+      - dev
+  pull_request:
+    branches:
+      - master
+      - dev
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Frontend - Deploy package
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      name: Check out repository
+
+    - run: |
+        # Current/target branches' names
+        branch=${GITHUB_REF#refs/heads/}
+        target_branch="${{ github.event.pull_request.base.ref }}"
+
+        # Extension Publishers
+        dev_publisher=retroextcd-rc
+        pr_publisher=retroextcd-prval
+
+        # Extension Prefix
+        # Remove all forward slashes to use for Extension Id
+        dev_ext_prefix=${branch//\//}
+        pr_branch=${{ github.event.pull_request.head.ref }}
+        if [[ "$pr_branch" ]]; then
+          pr_ext_prefix=${pr_branch//\//}
+        fi
+
+        # Test Organizations
+        dev_test_organization=retroextcd-rc
```

Comment:
@phongcao Are you planning to update the draft PR with details on Azure Organization that was set up? This will be good documentation for Engin/other OSS maintainers

Reply:
Currently we're having task #1570 (assigned to @argoldbe + @msa2984) to update the docs. I'm not sure whether that is something added to the draft design document or other files. Maybe we should all get together and discuss.